demo/DimImagesTransform.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang=en>
+
+  <head>
+    <meta charset=utf-8>
+    <title>DimImagesTransform</title>
+    <link href=../build/wikimedia-page-library-transform.css rel=stylesheet>
+    <style>
+      body, img, input { margin: 14px; }
+    </style>
+  </head>
+
+  <body>
+
+    <input type='checkbox' onclick='dimmingToggle()' id='toggleCheckbox'> Dim images
+    <br>
+    <script>
+      document.getElementById('toggleCheckbox').addEventListener('click', () => {
+        /* global pagelib */
+        pagelib.DimImagesTransform.dim(
```

Comment:
Sorry to comment on this again. It seems like this code would more naturally appear in the script block at the bottom of the file. pagelib is not available until the script below it runs.